### DISCUSSION TITLE
Add ability to snooze auto-assignment for team members

### DISCUSSION THREAD

**Discussion by brendonjwong** [community/community#62327] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

We currently use Github's automatic round robin assignment for code reviews on my team. When someone is away on vacation for 7+ days or on leave for any other reason, we manually add them to the list of `Never assign certain team members` and then have to remember to undo that change when they return so that PRs do not get stuck waiting for a review in their absence.

I'd love if we could set a date and time to start assigning PRs to them again.

**Comment by h-anders-unext** [comment]
We would love to have the same feature too.
Preferable some way for a single team member to quickly set themselves unavailable for a period of time.
